TYPO3.TYPO3CR/Resources/Private/Schema/NodeTypes.schema.yaml

```diff
@@ -38,6 +38,7 @@ additionalProperties:
         type: dictionary
         additionalProperties: FALSE
         properties:
+          'label': { type: string, description: "Human-readable label for this child node." }
```

Comment:
should be more equal to `An EEL expression for generating a node's of this node type's label. Alternatively an array with the key generatorClass and optional options.`